app/src/test/java/org/mozilla/fenix/share/ShareControllerTest.kt

```diff
@@ -272,7 +275,7 @@ class ShareControllerTest {
 
     @Test
     fun `getShareText should respect concatenate shared tabs urls`() {
-        assertEquals(textToShare, controller.getShareText())
+        assertEquals(textToShare, buildController(data = shareData).getShareText())
```

Comment:
I'm guessing that the answer to above q is "no" based on this 😛

Reply:
Yeah, the point of `buildController` is to make it easier to override in some tests. We could always build a plain controller and sometimes make a second one, but I felt that it wasn't a lot of code to just call the plain function in some locations.